documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -46,6 +46,16 @@ Limits can be overridden by xref:con-configuring-client-quotas-str[user-specific
 * CPU utilization limits for each client can be set as a percentage of the network threads and I/O threads on a per-broker basis.
 * The number of concurrent partition creation and deletion operations (mutations) allowed per second can be set on a per-broker basis.
 
+.Impact on internal users
+When using the default Kafka quotas plugin, quotas are applied to all users, including internal users such as the Topic Operator or Cruise Control. 
+This means that internal users can hit the set quotas, potentially affecting their operations. 
```

Comment:
```suggestion
When using the default Kafka quotas plugin, quotas are applied to all users. 
Users include internal users such as the Topic Operator or Cruise Control, potentially affecting their operations. 
```
maybe???

Reply:
I've updated along these lines. I like that clearer leading sentence.